mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/252 dialog up down message

Closes #252 

To test:
- Run Training/Stop Worms scenario
- Follow help tips until a message dialog pops up
- Check server process command prompt to see dialog_up and dialog_down messages are being sent properly
- Check on server messages when confirmation dialog pops up later in scenario
